hypervisor/arch/x86/ept.c

```diff
@@ -103,19 +103,12 @@ uint64_t gpa2hpa(struct vm *vm, uint64_t gpa)
 	return local_gpa2hpa(vm, gpa, NULL);
 }
 
+/**
+ * @pre: only SOS will use hpa2gpa and they're identical mapping in SOS.
+ */
 uint64_t hpa2gpa(struct vm *vm, uint64_t hpa)
 {
-	uint64_t *pgentry, pg_size = 0UL;
```

Comment:
let us change this API to sos_hpa2gpa(uint64_t hpa).

Reply:
Done